### ISSUE TITLE
equal min and max rpm creates error and does not allow any changes

### ISSUE THREAD

**Issue by pporzer** [crystalidea/macs-fan-control#123]
Probably due to a previous input error the minimum and maximum rpm are identical for my disk drive on an iMac. The program does not allow me to change from automatic to custom mode and issues an error message describing the fact that both limits are equal. How can I change these values?

![screen shot 2018-10-10 at 16 30 43](https://user-images.githubusercontent.com/1548009/46743683-e5b23180-cca9-11e8-96fb-0b9cdadeb967.png)

Macs Fan Control 1.4.12
iMac11,1
CPU: Intel(R) Core(TM) i5 CPU 750 @ 2.67GHz [100]
GPU: ATI Radeon HD 4850
macOS 10.13.6

Fans:
ODD-min{1000}curr{2225}max{3800}mode{2,TO0p,33,43}
HDD-min{1100}curr{1647}max{1100}mode{0}
CPU-min{940}curr{1616}max{4200}mode{2,TC0C,34,70}

TempSensors:
TA0P () - 20.9492, 20.9492
TA0V () - 21.0508, 21.0508
TA0p (Ambient) - 26.5, 26.5
TA1v () - 26.5391, 26.5391
TC0C (CPU Core 1) - 41.4219, 41.4219
TC0D () - 129, 129
TC0H (CPU Heatsink) - 39.375, 39.375
TC0P () - 129, 129
TC1C (CPU Core 2) - 41, 41
TG0D (GPU Diode) - 48.25, 48.25
TG0H (GPU Heatsink) - 45.875, 45.875
TG0p (GPU Proximity) - 48, 48
TH0O () - 9, 9
TH1O () - 9, 9
TL0P () - 128, 128
TL0V () - 128, 128
TL0p () - 128, 128
TL1V () - 128, 128
TL2V () - 128, 128
TLAV () - 128, 128
TLBV () - 128, 128
TLCV () - 128, 128
TMCD () - 0, 0
TO0P () - 37.125, 37.125
TO0p (Optical Drive) - 38.125, 38.125
TPCD (Platform Controller Hub Die) - 59, 59
TS0V () - 38.3789, 38.3789
TS2P () - 39.5, 39.5
TS2V () - 37.6797, 37.6797
TS2p () - 39.5, 39.5
Tm0P () - 41.75, 41.75
Tm0p () - 41.75, 41.75
Tp1P () - 48.125, 48.125
Tp2H () - 50.375, 50.375
Tp3H (PSU Primary) - 48.125, 48.125

Disk drives:
Hitachi HDE721010SLA330 - 51

**Comment by kleuter** [comment]
Please try to reset SMC

**Comment by pporzer** [comment]
Did reset the SMC, no change! Sorry! 
Peter

**Comment by pporzer** [comment]
Tried your recommended command line - min and max values stay the same and  are equal. The program smc-write seems to be a derivative of the program smc - it uses a very similar or better identical syntax -but with limited capabilities.
The critical fan is now in auto-mode and does a good job in keeping reasonable temps, interestingly it always runs beyond the min max values.
Is there anything I can do or should I just leave it as is?

**Comment by kleuter** [comment]
If the temps are OK, yes. Normally the fan should be controlled fine with the app

**Comment by sriasana** [comment]
Okay, when this error appears, other fan-managing apps may be the culprit, or hardware issues. I thought I had all other apps not only closed but also uninstalled with CleanMyMac that eliminates also Binaries and other support files. After acting stupid and resetting SMC and P RAM numerous times, I remembered iStat. Well the sensors were turned off, but when I clicked on Edit Fan Rules, the loud fan slowed down for the first time in days. Thus, for the time being I am using iStat for fan control with success.
